packages/shared/src/components/cards/ActionsButtons/ActionButtons.module.css

```diff
@@ -0,0 +1,127 @@
+.upvotes {
```

Comment:
They do? I had no idea

Reply:
Should I ask them for a Lottie JSON or we can keep the CSS version? 👀 
@nensidosari @rebelchris

Reply:
I'm totally fine with both, your animation looks great, and since it's contained in one file it's easy to access/manage anyways even if we want to remove and replace with the new thing. 

Would though suggest u do pop the question to design so that we 100% sure they use Lottie and if we can use it as well and be aware for this and any future implementation.

Reply:
I think let's leave it for now as is.
Asked here: https://dailydotdev.slack.com/archives/C0381BQUEN9/p1720513195733419